docs/developer/base/openmetrics.md

```diff
@@ -48,4 +48,20 @@ For complete documentation on every option, see the associated templates for the
 ## Legacy
 
 This OpenMetrics implementation is the updated version of the original Prometheus/OpenMetrics implementation. 
-The [docs for the deprecated implementation](../legacy/prometheus.md) are still available as a reference.
\ No newline at end of file
+The [docs for the deprecated implementation](../legacy/prometheus.md) are still available as a reference.
+
+### Config changes between OpenMetrics V1 and V2
+There were config option changes between OpenMetrics V1 and V2, so please check if any updated OpenMetrics instances use deprecated options and update accordingly.
+
+Note that the `type_overrides` option is incorporated in the `metrics` option now.
+
+| OpenMetricsV1               | OpenMetricsV2                        |
+|-----------------------------|--------------------------------------|
+| `ignore_metrics`            | `exclude_metrics`                    |
+| `prometheus_metrics_prefix` | `raw_metric_prefix`                  |
+| `health_service_check`      | `enable_health_service_check`        |
+| `labels_mapper`             | `labels_rename`                      |
+| `label_joins`               | `share_labels`                       |
```

Comment:
we should elaborate the additional features in `share_labels` and `metrics` because it's not just a 1:1 rename.